package.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>fastrtps</name>
+  <version>2.0.0</version>
```

Comment:
Right now is 2.0.1, I'm unsure about the configure.ac in master though

Reply:
the configure.ac from the master repo sais 2.0.0